v22.1/changefeed-for.md

```diff
@@ -54,6 +54,7 @@ Option | Value | Description
 `resolved` | [`INTERVAL`](interval.html) | Emits [resolved timestamp](use-changefeeds.html#resolved-def) events for the changefeed. Resolved timestamp events do not emit until all ranges in the changefeed have progressed to a specific point in time. <br><br>Set an optional minimal duration between emitting resolved timestamps. Example: `resolved='10s'`. This option will **only** emit a resolved timestamp event if the timestamp has advanced and at least the optional duration has elapsed. If unspecified, all resolved timestamps are emitted as the high-water mark advances.
 `envelope` | `key_only` / `row` | Use `key_only` to emit only the key and no value, which is faster if you only want to know when the key changes.<br><br>Default: `envelope=row`
 `cursor` | [Timestamp](as-of-system-time.html#parameters)  | Emits any changes after the given timestamp, but does not output the current state of the table first. If `cursor` is not specified, the changefeed starts by doing a consistent scan of all the watched rows and emits the current value, then moves to emitting any changes that happen after the scan.<br><br>`cursor` can be used to start a new changefeed where a previous changefeed ended.<br><br>Example: `CURSOR=1536242855577149065.0000000000`
+`end_time` | [Timestamp](as-of-system-time.html#parameters) | <span class="version-tag">New in v22.1:</span> Indicate the time when a changefeed should stop. Provide a future timestamp to `end_time` in number of nanoseconds since the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time). For example, `end_time="1654117200000000"`. At the specified time, the changefeed will stop running and return a `successful` status.
```

Comment:
It could be better to word this as something more like
> Indicate the time for which the changefeed will emit all events up to the given timestamp and then complete with a `successful` status.

Since the changefeed job itself could potentially still be running after the `end_time` passes if the timestamp of the actual events its emitting are still lagging behind.

Reply:
Would "Indicate the timestamp" rather than "a timestamp" be fine?  Very minor but personally "the" gives me more confidence that this is the event timestamp, since it must be referring to the only timestamp we've ever talked about so far, while "a" kind of reminds me that this could have multiple meanings and I feel a little less certain.